pkg/yurtmanager/webhook/util/configuration/configuration.go

```diff
@@ -75,6 +81,40 @@ func Ensure(kubeClient clientset.Interface, handlers map[string]struct{}, caBund
 
 		mutatingWHs = append(mutatingWHs, *wh)
 	}
+
+	// add additional deployment mutating webhook
+	wh := &admissionregistrationv1.MutatingWebhook{}
```

Comment:
please use `failPolicy=Ignore` instead of modifying webhook configuration.

Reply:
fix